Gemfile

```diff
@@ -12,8 +12,10 @@ end
 gem 'rails', '~> 5.0.1'
 gem 'rails-controller-testing'
 
-gem 'sprockets', '~> 3.6.3'
-gem 'sprockets-rails', '~> 3.1.1'
+# Compile Sprockets assets concurrently in `assets:precompile`.
+# Ref: https://github.com/rails/sprockets/pull/469
+gem 'sprockets', github: 'wjordan/sprockets', ref: 'concurrent_asset_bundle_3.x'
```

Comment:
yeah, this is my standard workflow for github-based patches tracking upstream. I have always defaulted to fork projects into my own GitHub account rather than into `code-dot-org`- though there isn't much difference between the two, I would probably only do the latter if it's a long-lived project fork we intend to collectively maintain internally as an organization (like we did with johnny-five or blockly). happy to change this if we want to standardize on another workflow though